static/app/views/performance/browser/resources/resourceSummaryPage/sampleImages.tsx

```diff
@@ -134,7 +134,7 @@ function SampleImagesChartPanelBody(props: {
             src = url.href;
           } catch {
             Sentry.captureException(new Error('Invalid URL'), {
-              data: {
+              tags: {
                 [SPAN_DESCRIPTION]: resource[SPAN_DESCRIPTION],
```

Comment:
The cardinality on this is pretty high, and will vary based on users quite a bit (we don't need to index them).

Let's do `contexts` instead https://docs.sentry.io/platforms/javascript/enriching-events/context/